module/canvas/detection-modes/detect-poison-and-disease.mjs

```diff
@@ -0,0 +1,53 @@
+/**
+ * The detection mode for Detect Poison and Disease.
+ */
+export class DetectionModeDetectPoisonAndDisease extends DetectionMode {
+  constructor() {
+    super({
+      id: "detectPoisonAndDisease",
+      label: "DND5E.DetectPoisonAndDisease",
+      type: DetectionMode.DETECTION_TYPES.OTHER,
+      walls: false
+    });
+  }
+
+  /** @override */
+  static getDetectionFilter() {
+    return this._detectionFilter ??= GlowOverlayFilter.create({
+      glowColor: [0, 1, 0, 1]
```

Comment:
It might be a good idea to allow the glow color (and other parameters?) to be modified by modules and/or settings (e.g. for those with color blindness).